MergerService/Utils/TaskUtils.cs

```diff
@@ -1,46 +1,209 @@
 using System.Diagnostics;
+using System.Text;
+using MergerLogic.Utils;
+using MergerService.Controllers;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
 
 namespace MergerService.Utils
 {
     // TODO: rename all utils to clients
-    public class TaskUtils
+    public class TaskUtils : ITaskUtils
     {
+        private IHttpRequestUtils _httpClient;
+        private IConfigurationManager _configuration;
         private ILogger _logger;
         private ActivitySource _activitySource;
 
-        public TaskUtils(ILogger<TaskUtils> logger, ActivitySource activitySource)
+        public TaskUtils(IConfigurationManager configuration, IHttpRequestUtils httpClient, ILogger<TaskUtils> logger, ActivitySource activitySource)
         {
+            this._httpClient = httpClient;
+            this._configuration = configuration;
             this._logger = logger;
             this._activitySource = activitySource;
-            //TODO: add tracing and logging
+            //TODO: add tracing
         }
 
         // TODO: add update progress method
-        public static string GetTask()
+        public MergeTask? GetTask(string jobType, string taskType)
         {
-            return @"{
-                ""Batches"": [
-                    {
-                        ""maxX"": 312254,
-                        ""maxY"": 85399,
-                        ""minX"": 312239,
-                        ""minY"": 85365,
-                        ""zoom"": 18
-                    }
-                ],
-                ""Sources"": [
-                    {
-                        ""Path"": ""output/temp"",
-                        ""Type"": ""FS"",
-                        ""Origin"": ""LL"",
-                        ""Grid"": ""2X1""
-                    },
-                    {
-                        ""Path"": ""area2.gpkg"",
-                        ""Type"": ""GPKG""
-                    }
-                ]
-            }";
+            string baseUrl = this._configuration.GetConfiguration("TASK", "jobManagerUrl");
+            string url = $"{baseUrl}/tasks/{jobType}/{taskType}/startPending";
+            string taskData = this._httpClient.PostDataString(url);
+
```

Comment:
You need to call heartbit manager with start() method

Reply:
Opened and added.

Reply:
https://mapcolonies.atlassian.net/browse/MAPCO-2382